src/main/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImpl.java

```diff
@@ -44,17 +43,22 @@
 public class DataAnonymisationServiceImpl implements DataAnonymisationService {
 
     private final AuditApi auditApi;
-    private final UserIdentity userIdentity;
 
     private final CurrentTimeHelper currentTimeHelper;
     private final ExternalOutboundDataStoreDeleter outboundDataStoreDeleter;
-    private final CaseRepository caseRepository;
     private final TransformedMediaRepository transformedMediaRepository;
     private final TransientObjectDirectoryRepository transientObjectDirectoryRepository;
     private final LogApi logApi;
+    private final CaseService caseService;
+
 
     @Override
-    public void anonymizeCourtCaseEntity(UserAccountEntity userAccount, CourtCaseEntity courtCase) {
+    @Transactional
+    public void anonymizeCourtCaseById(UserAccountEntity userAccount, Integer courtCaseId) {
```

Comment:
Can we use the English spelling of anonymise and maybe update anywhere that uses the American version to be consistent?

Reply:
I think this is fixed in another PR that has not yet been merged but can check

Reply:
Doesn't look like this has updated